lib/board.rb

```diff
@@ -0,0 +1,65 @@
+require 'pry'
+
+class Board
+
+  attr_accessor :cells
+
+  def initialize(cells = Array.new(9, " "))
+    @cells = cells 
+  end 
+
+  def reset!
+    @cells = Array.new(9, " ")
+  end 
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end 
+
+  def position(input)
+    # binding.pry
+      if input.to_i > 0 && input.to_i < 10
+      cells[input.to_i-1]
+    else
+      false 
+    end 
+  end
+
+  def full?
+    if cells.include?(" ")
+      false
+    else 
+      true
+    end 
+  end 
```

Comment:
Use the implicit boolean return value of the expression instead of adding an if statement. You can use the bang operator `!` to negate or revers the results. 
```
def full?
  !cells.include?(" ")
end
```